openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -257,6 +262,75 @@ def check_permissions(self, instance: dict[str, Any]) -> None:
         self.check_group_F(actual_group_fields["F"])
         self.check_group_G(actual_group_fields["G"])
 
+    def _meeting_admin_can_manage_non_admin(self) -> bool:
+        """
+        Checks if the requesting user has permissions to manage participants in all of requested users meetings.
+        Also checks if the requesting user has meeting admin rights and the requested user doesn't.
+        Returns true if permissions are given. False if not. Raises no Exceptions.
+        """
+        if not self.instance_id:
+            return False
+        b_user = self.datastore.get(
+            fqid_from_collection_and_id("user", self.instance_id),
```

Comment:
I don't think `instance_id` needs to be an instance attribute, just pass it on as a parameter

Reply:
Done